sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/BasicWriteStatsTracker.scala

```diff
@@ -223,6 +278,9 @@ class BasicWriteJobStatsTracker(
 
     val executionId = sparkContext.getLocalProperty(SQLExecution.EXECUTION_ID_KEY)
     SQLMetrics.postDriverMetricUpdates(sparkContext, executionId, driverSideMetrics.values.toList)
+
+    SQLPartitionMetrics.postDriverMetricUpdates(sparkContext, executionId,
```

Comment:
These new partition metrics are not shown in the Spark UI, but a listener event that third-party libraries can consume?

Reply:
The existing collected metrics through `SQLMetrics` are limited.  `SQLMetic` extends `AccumulatorV2[Long, Long]` so I didn't see a clear way for the V1 code to use a more complex AccumulatorV2 of a map.  I wanted to start collecting the data, and look to improve the implementation over time.